src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,27 @@
+.stars {
+  display: flex;
+  width: 96px;
```

Comment:
Avoid setting a fixed width for the `.stars` block. The width should be determined by the content and the space between stars. This will make your component more flexible and reusable.